### DISCUSSION TITLE
Error while Installing

### DISCUSSION THREAD

**Discussion by Praveenstein** [apache/streampipes#885] [Q&A] [answered]
When i tried installing using docker-compose up -d, it says (On Windows 10)

=> ERROR [3/3] COPY target/streampipes-extensions-all-jvm.jar  /streampipes-extensions-all-jvm.jar                                                                                                                                     0.0s
------
 > [3/3] COPY target/streampipes-extensions-all-jvm.jar  /streampipes-extensions-all-jvm.jar:
------
failed to compute cache key: failed to walk /var/lib/docker/tmp/buildkit-mount979314341/target: lstat /var/lib/docker/tmp/buildkit-mount979314341/target: no such file or directory
ERROR: Service 'extensions-all-jvm' failed to build : Build failed

Please help me out

![strempipe_error_1](https://user-images.githubusercontent.com/45847217/209288539-f14a7d76-f5af-4c0b-8e95-d79d4d83b1ff.png)

**Comment by Praveenstein** [comment]
Ok, it seems like i was in the wrong directory, anyways i'm getting new error:

Creating network "streampipes_spnet" with driver "bridge"
ERROR: Pool overlaps with other one on this address space

![Screenshot 2022-12-23 125344](https://user-images.githubusercontent.com/45847217/209291365-da25eaf0-d890-484f-a8c2-28ef8acffce8.png)

**Comment by tenthe** [comment, marked as answer]
Hi @Praveenstein,
thanks for trying out StreamPipes.

It seems that the first command already created the docker network `streampipes_spnet`. Can you try to run `docker-compose down` and then `docker compose up -d` again?
If this does not work please check if there is a docker network named **spnet** with the command `docker network ls`. If it is there you can remove it with `docker network rm #NETWORK_ID`.

Please let us know if it worked. 
Also feel free to ask if you have any other problems or improvement suggestions, we are happy to help

Cheers,
Philipp

**Reply by Praveenstein** [reply]
Thanks, worked like a charm!
